argo_rollouts/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,16 @@
+name: Argo Rollouts
+files:
+- name: argo_rollouts.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.required: true
```

Comment:
```suggestion
```

nit - this is already the case https://github.com/DataDog/integrations-core/blob/84115ad97c0aed62c578dfb46411f2a671447b3b/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml#L1-L2